index.html

```diff
@@ -813,39 +813,44 @@ <h2>
         "!SERVICE-WORKERS-1#scope-match-algorithm">Service Workers</a>. To
         avoid unexpected behavior, use a scope ending in a <code>/</code>.
       </div>
-      <div class="issue" title="🐒 patch">
+      <p>
+        If the <a>application context</a>'s <a
+        data-cite="!HTML#active-document">active document</a>'s <a
+        data-cite="!DOM#concept-document-url">URL</a> is not <a>within scope</a>
+        of the navigation scope of the <a>application context</a>'s manifest,
+        the user agent SHOULD show prominent UI indicating the <a
+        data-cite="!DOM#concept-document-url">document URL</a>, or
+        at least its <a>origin</a>, including whether it is served over a secure
+        connection. This UI SHOULD differ from any UI used when the <a
+        data-cite="!DOM#concept-document-url">document URL</a> is <a>within
+        scope</a>, in order to alert the user of the off-scope navigation.
+      </p>
+      <div class="note">
+        <p>
+          Nothing prevents an <a>application context</a> from navigating to a
+          <a>URL</a> that is outside of the application's <a>navigation
+          scope</a>, while still having the <a>manifest</a> <a>applied</a> to
+          it.
+        </p>
         <p>
-          Enforcing the navigation scope depends on [[!HTML]]'s navigate
-          algorithm. As such, the following algorithm monkey patches [[!HTML]].
-          <a href="https://www.w3.org/Bugs/Public/show_bug.cgi?id=27653">Bug
-          27653</a> has been filed to address this.
+          Unlike previous versions of this specification, user agents are no
+          longer required or allowed to block off-scope navigations, or open
+          them in a new <a>top-level browsing context</a>. This practice broke
+          a lot of sites that navigate to a URL on another origin (e.g., to
```

Comment:
"to a URL on another origin" -> "to an off-scope URL" (since scope can be narrower than an origin)